api/src/server.ts

```diff
@@ -155,11 +157,28 @@ export async function startServer(): Promise<void> {
 	const server = await createServer();
 
 	const host = env['HOST'] as string;
-	const port = parseInt(env['PORT'] as string);
+	const path = env['UNIX_SOCKET_PATH'] as string | undefined;
+	const port = env['PORT'] as string | undefined;
+
+	let listenOptions: ListenOptions = {
+		path,
+	};
+
+	if (port || !path) {
+		// if port is present we ignore socket path
+		listenOptions = {
+			host,
+			port: parseInt(port || '8055'),
+		};
+	}
```

Comment:
Here be opinions!

I'd prefer it if we could do:

```
let listenOptions: ListenOptions;

if (path) {
  listenOptions = {...};
} else {
  listenOptions = {...};
}
```

Reply:
Same here, I think this is a simpler and better to read code 👍